src/connectMultireducer.js

```diff
@@ -28,22 +30,15 @@ export default function connectMultireducer(mapStateToProps, actions = {}) {
       generateConnectedComponent({multireducerKey}) {
         this.ConnectedComponent =
           connect(
-            (state, ownProps) => {
-              const multireducerKeys = Object.keys(state.multireducer);
-              if (!multireducerKeys.filter(key => key === multireducerKey)[0]) {
-                throw new Error(`No state for multireducer key "${multireducerKey}". You initialized multireducer with "${multireducerKeys.join(', ')}".`);
-              }
-              const slice = state.multireducer[multireducerKey];
-              return mapStateToProps ? mapStateToProps(slice, ownProps) : slice;
-            },
-            multireducerBind(actions, multireducerKey)
+            wrapMapStateToProps(mapStateToProps, multireducerKey),
+            wrapMapDispatchToProps(mapDispatchToProps, multireducerKey),
+            ...rest
```

Comment:
@jsdmc Somewhere in here, you lost the [BUT ONE IMPORTANT DIFFERENCE](https://github.com/erikras/multireducer#-mapstatetopropskey-state-ownprops--function-optional) logic that was slicing up the state to only the slice that the reducer wants. Does that make sense?